kubernetes/redpanda/templates/statefulset.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "redpanda.name" . }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "redpanda.name" . }}
+  serviceName: {{ template "redpanda.name" . }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "redpanda.name" . }} 
+    spec:
+      initContainers:
+        - name: {{ template "redpanda.name" . }}-configurator
+          image: {{ .Values.image.repository }}
+          command: ["/bin/sh", "-c"]
+          args:
+            - >
+              CONFIG=/tmp/config/redpanda.yaml;
+              NODE_ID=${HOSTNAME##*-};
+              SERVICE_NAME={{ template "redpanda.name" . }}-$NODE_ID;
+              cp /tmp/base-config/redpanda.yaml $CONFIG;
+              rpk --config $CONFIG config set redpanda.node_id $NODE_ID;
+              if [ "$NODE_ID" = "0" ]; then
+                rpk --config $CONFIG config set redpanda.seed_servers '[]' --format yaml;
```

Comment:
Is it OK for this node to be restarted without seeds? @senior7515 

It seems to be OK for nodes to come up with themselves as seeds https://github.com/vectorizedio/redpanda/blob/7f5805125d26c61655c444da5e87fa4d82d239b4/src/v/cluster/members_manager.cc#L257